torchtext/prototype/models/t5/bundler.py

```diff
@@ -133,6 +135,129 @@ def build_model(
 
         return model
 
+    @staticmethod
+    def build_model_from_huggingface_ckpt(
+        ckpt_path: Union[str, os.PathLike],
+        *,
+        freeze_model: bool = False,
+        strict: bool = True,
+    ) -> T5Model:
+        config_path = f"{ckpt_path}/config.json"
+        model_path = f"{ckpt_path}/pytorch_model.bin"
+
+        assert os.path.exists(model_path), f"No PyTorch model found at {model_path}"
+
+        with open(config_path, "r") as handle:
```

Comment:
This is OSS compatible, but not internally compatible. Coming in a follow up commit.